apps/dashboard/templates/meinberlin_dashboard/base_user.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Dashboard" %}{% endblock %}
+
+{% block content %}
+
+<div class="l-wrapper">
+    <div class="menu-layout">
+
+        <div class="menu-layout__menu">
+            <div class="dropdown">
```

Comment:
i am not sure why the div wrapping the username has a dropdown class. 
did you mean to make the whole account settings a dropdown?